spark3/src/main/java/org/apache/iceberg/spark/FileRewriteCoordinator.java

```diff
@@ -71,6 +72,8 @@ public void commitRewrite(Table table, Set<String> fileSetIDs) {
     table.newRewrite()
         .rewriteFiles(rewrittenDataFiles, newDataFiles)
         .commit();
+
```

Comment:
I think we need to remove redundant logic from this class now. Can be done in a separate PR to simplify the review.
We should probably rename it too as it is no longer a commit coordinator.

Reply:
We can probably align the naming with the task set manager or whatever we call it on the read side.

Reply:
Yeah i'll do another pr after this which prunes this class